tests/rest/client/v1/test_login.py

```diff
@@ -0,0 +1,110 @@
+import json
+
+from synapse.rest.client.v1 import admin, login
+
+from tests import unittest
+
+
+class LoginRestServletTestCase(unittest.HomeserverTestCase):
+
+    servlets = [
+        admin.register_servlets,
+        login.register_servlets,
+    ]
+
+    def make_homeserver(self, reactor, clock):
+
+        self.url = b"/_matrix/client/r0/login"
```

Comment:
make this a global constant or something?

Reply:
As asked in #synapse-dev, why do that here while other tests (register, capabilities, etc) have a `self.url` defined this way?

Reply:
Gah, I missed the response there. Will change that then.